hello/src/main/java/dupa/Dupa.java

```diff
@@ -0,0 +1,34 @@
+package dupa;
+
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.eclipse.jetty.servlet.ServletHolder;
+import org.glassfish.jersey.server.ResourceConfig;
+import org.glassfish.jersey.servlet.ServletContainer;
+
+/**
+ * Created by grzes on 20.05.17.
```

Comment:
Set in idea not to add this comments as such a things are in git history. 
Also remove this one.